VirtoCommerce.Storefront/Domain/ImageProcessing/ImagekitImageProcessor.cs

```diff
@@ -0,0 +1,32 @@
+using System.Linq;
+using Microsoft.Extensions.Options;
+using VirtoCommerce.Storefront.Model.StaticContent;
+
+namespace VirtoCommerce.Storefront.Domain.ImageProcessing
+{
+    public class ImagekitImageProcessor : IImageProcessor
+    {
+        private readonly string _serviceUrl;
+
+        public ImagekitImageProcessor(IOptions<ImageProcessorOptions> options)
+        {
+            _serviceUrl = options.Value.ServiceUrl.TrimEnd('/');
+        }
+        public string ResolveUrl(string inputUrl, int width, int height)
```

Comment:
width and height can be optional and it will be better to do them nullable

Reply:
yes